packages/phoenix-otel/src/phoenix/otel/otel.py

```diff
@@ -192,3 +237,45 @@ def _maybe_grpc_endpoint(parsed_endpoint: ParseResult) -> bool:
     if not parsed_endpoint.path and parsed_endpoint.port == 4317:
         return True
     return False
+
+
+def _exporter_transport(exporter: SpanExporter) -> str:
+    if isinstance(exporter, _HTTPSpanExporter):
+        return "HTTP"
+    if isinstance(exporter, _GRPCSpanExporter):
+        return "gRPC"
+    else:
+        return exporter.__class__.__name__
+
+
+def _normalize_headers(headers: Union[List[Tuple[str, str]], Dict[str, str]]) -> Dict[str, str]:
```

Comment:
Normalize is probably not the right word. It's just for printing so maybe something like https://github.com/Arize-ai/phoenix/blob/ce00d50930589f97d738f05f39d3d7b3c145e967/src/phoenix/db/engines.py#L32